src/components/calculator.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import '../index.css';
+
+function Calculator() {
+  /* eslint no-eval: 0 */
+
+  const [input, setInput] = useState('');
+  const calcBtns = [];
+  [9, 8, 7, 6, 5, 4, 3, 2, 1, 0, '.', '%'].forEach((item) => {
+    calcBtns.push(
+      <button
+        onClick={(e) => {
+          setInput(input + e.target.value);
+        }}
+        value={item}
+        key={item}
+      >
+        {' '}
+        {item}
+      </button>,
+    );
+  });
+
+  return (
+    <div className="wrapper">
+      {' '}
+      <div className="show-input">{input}</div>
+      <div className="digits flex">{calcBtns}</div>
+      <div className="modifiers subgrid">
+        {/* clear button */}
+
+        <button onClick={() => setInput(input.substr(0, input.length - 1))}>
+          Clear
+        </button>
+
+        {/* clear all */}
+        <button onClick={() => setInput('')} value="">
+          AC
+        </button>
+      </div>
+      <div className="operations subgrid">
+        {/* add button */}
+        <button onClick={(e) => setInput(input + e.target.value)} value="+">
+          +
+        </button>
+
+        {/* minus btn */}
+        <button onClick={(e) => setInput(input + e.target.value)} value="-">
+          {' '}
+          -
```

Comment:
- [ ] For this project milestone, you are required to develop just the UI of the calculator; the math logic to make it work will be added in the next project. Please prioritize achieving the given UI design for this project milestone.